### PR TITLE
Add save command.

### PR DESCRIPTION
**Pull request by fennecdjay**

### CODE REVIEW

File: zpm.c [hunk 1]
```diff
@@ -399,6 +399,27 @@ int plugin_print_list() {
     return 0;
 }
 
+int plugin_print_script() {
+    char entry[PATH_MAX];
+    char* zpm_init = get_plugin_list_path();
+    FILE* store = fopen(zpm_init, "r");
+
+    if (!store) {
+        // migth need an error message here.
```
Comment (desyncr) [inline zpm.c]: printf("Could not open \"%s\". Check the file exists and can be read.", zpm_init);

**Review by desyncr — APPROVED** [review]
Add error message and tests for that message:

    $ zpm reset
    $ zpm save
    Could not open ".*". Check the file exists and can be read. (re)

File: zpm.c (hunk 1 above)
Comment (fennecdjay) [reply zpm.c]: Thank you.

**Comment by desyncr** [comment]
Awesome! Thanks again for the great work!

**Comment by fennecdjay** [comment]
My pleasure ;-)

**Pull request merged**